lib/rules/multiline-ternary.js

```diff
@@ -0,0 +1,16 @@
+/**
+ * @author dev1437
+ * See LICENSE file in root directory for full license.
+ */
+'use strict'
+
+// ------------------------------------------------------------------------------
+// Requirements
+// ------------------------------------------------------------------------------
+
+const { wrapCoreRule } = require('../utils')
+
+// eslint-disable-next-line no-invalid-meta, no-invalid-meta-docs-categories
+module.exports = wrapCoreRule('multiline-ternary', {
+  skipDynamicArguments: true
```

Comment:
I think it's probably a rule that can be applied in CSS v-bind as well.

```suggestion
  skipDynamicArguments: true,
  applyDocument: true
```

Additionally, could you add test cases to check if it works?

Reply:
Will this also include ternary statements in the `<script>` block? I think those should be excluded, as the core `multiline-ternary` rule will catch those already.

Also, please update the docs and/or description to mention `v-bind` in the `<style>` block.